wdl/qc_imputation.wdl

```diff
@@ -93,24 +112,39 @@ workflow qc_imputation {
         f=f, het_sd=batch_qc.het_sd_threshold[0], pi_hat_min_n=batch_qc.pi_hat_min_n_threshold[0], docker=docker
     }
 
+    # for chip qc, filter variants and samples in batches as per qc and merge batches
+    scatter (i in range(length(vcfs))) {
+        call filter_batch {
+            input: docker=docker, bed=vcf_to_bed.out_bed[i], joint_qc_exclude_variants=gather_snpstats_joint_qc.exclude_variants_joint,
+            batch_qc_exclude_variants=batch_qc.variants_exclude[i], exclude_samples=duplicates.allbatches_samples_exclude
+        }
+    }
+    scatter (chr in chrs) {
+        call merge_batches {
+            input: docker=docker, name=name+"_chr"+chr, chr=chr, beds=filter_batch.out_bed, bims=filter_batch.out_bim, fams=filter_batch.out_fam
+        }
+        call post_subset_sub.subset_samples as subset_samples_chip {
```

Comment:
hitto ku naurattaa ite taisin teha :) shouldn't this this post_subset_sub.subset_samples be outside the scatter? and again no list input but single vcf